File: If-Else Programming/002. Vowel or Consonant/C++/solution.cpp
#include<iostream>

using namespace std;

int main() {
    
    char a ;
  
    cout<<"Enter The Character You Want Check Vowel or Not\n";
    cin>>a;
    
    a = toupper(a);

    if(a=='A'||a=='E'||a=='I'||a=='O'||a=='U') {
        cout<< "Vowel";
    }
    else
    {
        cout<< "Consonant";
    }
    
    return 0;
}

File: If-Else Programming/002. Vowel or Consonant/Java/solution.java
import java.util.Scanner;
import java.io.*;
import java.lang.*;

public class Main
{
	public static void main(String args[])
    {
        char ch;
        Scanner sc = new Scanner(System.in);
 
        System.out.print("Enter an Alphabet To Check : \n");
        ch = sc.next().charAt(0);
        
        ch = Character.toUpperCase(ch);
 
        if(ch=='A' || ch=='E' || ch=='I' || ch=='O' || ch=='U')
        {
            System.out.print("Vowel");
        }
        else
        {
            System.out.print("Consonant");
        }
    }
}
